### PR TITLE
OSDOCS-9023:removes kuryr from networking docs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Scrubs Kuryr from networking docs

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9023
https://issues.redhat.com/browse/OSDOCS-9014

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @yakov103 
[PR](https://github.com/openshift/openshift-docs/pull/69905) with most of the content removed
https://github.com/openshift/openshift-docs/pull/69170/files#diff-01edc9e8a6e10d17824b95106e9a56fcda03474d7f9781026306f38c190dd155L23

### CODE REVIEW

File: networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
Comment (dulek) [inline networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc:1]: This file is included in _topic_map.yml, so I think it should be removed from there?

**Review by dulek — CHANGES_REQUESTED** [review]

**Review by jeana-redhat — APPROVED** [review]
looks good, double checked repo for `migrate-from-kuryr-sdn` and I don't see any missed xrefs or anything :+1: 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by yakov103 — APPROVED** [review]
/lgtm

**Pull request merged**